src/wazuh_modules/inventory_harvester/src/systemInventory/elements/networkProtocolElement.hpp

```diff
@@ -61,7 +61,14 @@ class NetworkProtocolElement final
         }
 
         element.data.network.dhcp = data->netProtoDhcp().compare("enabled") == 0 ? true : false;
-        element.data.network.gateway = data->netProtoGateway();
+
```

Comment:
note: I think in the future we should considered the case of multiple gateways, maybe changing the schema adding two gateways, or taking always the first position